attributes/default.rb

```diff
@@ -24,7 +24,7 @@
 # A resource provider must be defined for this version. For example, when set to 200, it will look
 # for the resource in OneviewCookbook::API200. When 300, it will look in OneviewCookbook::API300
 # See the libraries/resources directory for more info on supported API versions
-default['oneview']['api_version'] = 200
+default['oneview']['api_version'] = 300
```

Comment:
This will require a major version bump (breaking change). That's fine, just wanted to make note of it.